content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -0,0 +1,325 @@
+				</ul>
+				<p>Allowed Values:</p>
+				<ul>
+					<li><code>runtime/default</code></li>
+					<li>undefined (container annotation)</li>
+				</ul>
+			</td>
+		</tr>
+	</tbody>
+</table>
+
+## Policy Instantiation
+
+Decoupling policy definition from policy instantiation allows for a common understanding and
+consistent language of policies across clusters, independent of the underlying enforcement
+mechanism.
+
+As mechanisms mature, they will be defined below on a per-policy basis. The methods of enforcement
+of individual policies are not defined here.
+
+**PodSecurityPolicy**
+
+*   [Privileged](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/privileged-psp.yaml)
+*   [Baseline](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/baseline-psp.yaml)
+*   [Restricted](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/restricted-psp.yaml)
+
+## FAQ
+
+### Why isn't there a profile between privileged and default?
+
+The 3 profiles defined here have a clear linear progression from most secure (restricted) to least
+secure (privileged), and cover a broad set of workloads. Privileges required above the baseline
+policy are typically very application specific, so we do not offer a standard profile in this
+niche. This is not to say that the privileged profile should always be used in this case, but that
+policies in this space need to be defined on a case-by-case basis.
+
+We may reconsider this position in the future, should a clear need for other profiles arise.
+
+### What's the difference between a security policy and a security context?
+
+Security contexts are sets of conditions relating to how a container is configured that they want to
+be true in order for the container or pod to run. These security conditions are known as [Security
+Contexts](/docs/tasks/configure-pod-container/security-context/), and are implemented at a pod level
+by definition in a manifest.
+
+Security profiles however are control plane mechanisms to enforce standards relating to these
+security contexts. As of February 2020, the current native solution for enforcing these security
+policies is [Pod Security Policy](/docs/concepts/policy/pod-security-policy/) - a mechanism for centrally enforcing
+security control aspect standards across a cluster - which is currently in Beta. Other alternatives
+for the cluster-wide enforcement of security contexts are being developed, such as [OPA
+Gatekeeper](https://github.com/open-policy-agent/gatekeeper)
+
+### What profiles should I apply to my windows pods?
+
+Windows in Kubernetes has, by default, some limitations and differentiators from standard
+Linux-based workloads. Specifically, none of the PodSecurityContext fields [work on
+Windows](/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#v1-podsecuritycontext). As
```

Comment:
possible reword:
Specifically, the PodSecurityContext fields have no effect on Windows?